dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.anno-1.0.feature

```diff
@@ -3,11 +3,20 @@ symbolicName=com.ibm.websphere.appserver.anno-1.0
 IBM-API-Package: javax.annotation; type="spec", \
  javax.annotation.security; type="spec", \
  javax.annotation.sql; type="spec"
-IBM-SPI-Package: com.ibm.wsspi.anno.classsource, \
+IBM-SPI-Package: \
+ com.ibm.wsspi.anno.classsource, \
  com.ibm.wsspi.anno.info, \
  com.ibm.wsspi.anno.service, \
  com.ibm.wsspi.anno.targets, \
- com.ibm.wsspi.anno.util
+ com.ibm.wsspi.anno.util, \
+ com.ibm.ws.anno.classsource.specification, \
+ com.ibm.wsspi.annocache.classsource, \
+ com.ibm.wsspi.annocache.info, \
+ com.ibm.wsspi.annocache.service, \
+ com.ibm.wsspi.annocache.targets, \
+ com.ibm.wsspi.annocache.targets.cache, \
+ com.ibm.wsspi.annocache.util, \
+ com.ibm.ws.annocache.classsource.specification
```

Comment:
For beta should any of these be exposed, or should they just be exposed for when it is GA?

Reply:
I'll have to review to see why those two .specification packages are listed.  They are used by unit test code, but (to my recollection) not by any product code.

Reply:
Just because something is used by test or product code doesn't mean it needs to be listed as an SPI.  Internal components can communicate using packages that are not listed in the SPI list.  By the packages being exported they can be used by other bundles.  SPIs are meant for use by product extensions and user features.

Reply:
I've remove the two "specification" exports.

Reply:
You aren't exported the packages any longer, but you are still marking them as SPI.  That won't work.